src/main/kotlin/no/nav/dagpenger/regel/api/routing/V2BehovRoute.kt

```diff
@@ -0,0 +1,86 @@
+import io.ktor.response.header
+import io.ktor.response.respond
+import io.ktor.routing.Routing
+import io.ktor.routing.get
+import io.ktor.routing.post
+import io.ktor.routing.route
+import mu.KotlinLogging
+import no.nav.dagpenger.regel.api.BadRequestException
+import no.nav.dagpenger.regel.api.db.SubsumsjonStore
+import no.nav.dagpenger.regel.api.models.Behov
+import no.nav.dagpenger.regel.api.models.InntektsPeriode
+import no.nav.dagpenger.regel.api.models.Status
+import no.nav.dagpenger.regel.api.streams.DagpengerBehovProducer
+import java.time.LocalDate
+
+private val LOGGER = KotlinLogging.logger {}
+
+internal fun Routing.v2behov(store: SubsumsjonStore, producer: DagpengerBehovProducer) {
+    authenticate {
+        route("/v2") {
+            route("/behov") {
+                post {
+                    call.respond(HttpStatusCode.OK, "successfully authed v2/behov")
+                    /*mapRequestToV2Behov(call.receive()).apply {
+                        store.opprettBehov(this).also {
+                            producer.produceEvent(it)
+                        }.also {
+                            call.response.header(HttpHeaders.Location, "/behov/status/${it.behovId}")
+                            call.respond(HttpStatusCode.Accepted, V2StatusResponse("PENDING"))
+                        }.also {
+                            LOGGER.info("Produserte behov ${it.behovId} for intern id  ${it.behandlingsId} med beregningsdato ${it.beregningsDato}.")
+                        }
+                    }*/
+                }
+
+                route("/status") {
+                    get("/{behovId}") {
+                        val behovId = call.parameters["behovid"] ?: throw BadRequestException()
+
+                        when (val status = store.behovStatus(behovId)) {
+                            is Status.Done -> {
+                                call.response.header(HttpHeaders.Location, "/subsumsjon/${status.subsumsjonsId}")
+                                call.respond(HttpStatusCode.SeeOther)
+                            }
+                            is Status.Pending -> {
+                                call.respond(V2StatusResponse("PENDING"))
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+}
+
+private data class V2StatusResponse(val status: String)
+
+internal fun mapRequestToV2Behov(request: V2BehovRequest): Behov = Behov(
+    aktørId = request.aktorId,
+    vedtakId = request.vedtakId,
+    beregningsDato = request.beregningsdato,
+    harAvtjentVerneplikt = request.harAvtjentVerneplikt,
+    oppfyllerKravTilFangstOgFisk = request.oppfyllerKravTilFangstOgFisk,
+    bruktInntektsPeriode = request.bruktInntektsPeriode,
+    manueltGrunnlag = request.manueltGrunnlag,
+    antallBarn = request.antallBarn ?: 0,
+    inntektsId = request.inntektsId
+)
+
+internal data class V2BehovRequest(
+    val aktorId: String,
+    val vedtakId: Int,
```

Comment:
Skal ikke denne erstattes av "kontekst"?

Reply:
Jo, har bare ikke kommet så langt enda, ønsker å teste at kalkulator-api faktisk får snakket med v2/behov med x-api-key'en jeg har konfigurert

Reply:
Og er ikke helt sikker på hvordan vi ønsker at datastrukturen skal se ut. :P Antar vi beholder aktorId, og lar kontekst = { eksternId, context } ?

Reply:
kontekst = { eksternId, context } høres ut som en god plan.

En kunne jo bare laget en enkelt "ping" endepunkt for å teste api-nøkkel :) Så hadde en sluppet å ta i alt samtidig, sånn #walking-skeleton tankegang